### PR TITLE
chore!: Remove deprecated functions

### PR DESCRIPTION
**Pull request by gadenbuie**
Removes functions that were deprecated in Shiny v0.6.0 (panels, navset) and v0.6.1 (nav):

* `ui.panel_sidebar()`
* `ui.panel_main()`
* `ui.nav()`
* `ui.navset_pill_card()`
* `ui.navset_tab_card()`

### CODE REVIEW

File: shiny/express/ui/__init__.py [hunk 1]
```diff
@@ -21,35 +18,34 @@
...
     img,
     p,
     pre,
     span,
     strong,
-)
-
-from ...ui import (
-    fill,
-)
```
Comment (cpsievert) [inline shiny/express/ui/__init__.py:34]: How come this was removed?

File: shiny/express/ui/__init__.py [hunk 2]
```diff
@@ -58,76 +54,71 @@
...
+    remove_ui,
     update_accordion,
     update_accordion_panel,
-    update_sidebar,
     update_action_button,
     update_action_link,
     update_checkbox,
-    update_switch,
     update_checkbox_group,
-    update_radio_buttons,
```
Comment (cpsievert) [inline shiny/express/ui/__init__.py:87]: Seems you have a linter setting that is causing lots of changes here...can we avoid making a such large diff on this file?

Comment (gadenbuie) [reply shiny/express/ui/__init__.py:87]: yeah, sorry. I tried hard to avoid format-on-save but missed this one

Comment (gadenbuie) [reply shiny/express/ui/__init__.py]: @cpsievert It's fixed now

**Review by cpsievert — APPROVED** [review]

**Pull request merged**